_commands/tcpdump.md

```diff
@@ -0,0 +1,75 @@
+---
+---
+
+tcpdump
+-------
+
+tcpdump is a tool that allows you to view a dump of traffic on a network. By default, tcpdump will let you view the dump on your local network.
+You can use packet analyzing tools like Wireshark to analyze tcpdumps and pcap files.
+
+~~~ bash
+$ tcpdump 
+~~~
+
+<!--more-->
+
+### Useful Options / Examples
+
+Prerequisites
+- AS ROOT (to obtain root access, follow step 4 here: http://www.wikihow.com/Become-Root-in-Ubuntu) do the following steps:
```

Comment:
Just do `sudo -s` or `sudo su` to drop into a root prompt. You can actually link to our sudo article!

https://c4cs.github.io/commands/basics/sudo.html